portfolio/src/main/webapp/comment.js

```diff
@@ -1,28 +1,113 @@
 
-/** Creates an <li> element containing text. Use this to show a list of comments for now. */
+/** Creates an <p> element containing text. Use this to show a list of comments for now. */
 function createListElement(text) {
-  const liElement = document.createElement('li');
-  liElement.innerText = text;
+  const liElement = document.createElement('p');
+  console.log('comment text ' + text);
+  var nameComment = makeNameCommentArray(text);
+  liElement.innerText = nameComment[1];
+  liElement.style.paddingLeft = "15px";
+  liElement.style.fontSize = "15px";
```

Comment:
We are mixing presentation responsibilities with our javascript now. Consider using a css class for what a "comment list element" should have and then apply the class in the javascript rather than the style.

Here and below in line 24-26

Reply:
Ok, that makes sense. Done!